README.md

```diff
@@ -4,3 +4,22 @@
 
 1. Install [rustup](https://rustup.rs/).
 2. Install Ruby using your preferred package manager.
+
+### Release process
+
+To release a new version of SDK:
+1. Make sure that version strings have been updated:
+   - Eppo core: [eppo_core/Cargo.toml](eppo_core/Cargo.toml)
```

Comment:
My understanding is this should be treated with its own semver tree; for example breaking changes in eppo core might not be breaking changes up stream?

Reply:
Yes, all components have independent versioning